app/views/application/_subnavigation.html.erb

```diff
@@ -8,7 +8,6 @@
   <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-0 govuk-!-padding-right-0 flex-end">
     <nav class="govuk-body govuk-!-margin-bottom-0 ">
     <% if current_user.admin? %>
-      <%= link_to "Organisations", organisations_path, class: active_tab(organisations_path) %>
```

Comment:
Why have you removed this?

Reply:
Figured since the homepage redirects to the organisation path, that tab would not be needed.

Reply:
But then you wouldn't be able to get back to it. That aside I think its also important enough to be a subnav item.